src/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,149 @@
+import {
+  cloneElement,
+  KeyboardEvent,
+  ReactElement,
+  ReactNode,
+  useState
+} from 'react';
+
+import castArray from 'lodash/castArray';
+
+import Box from '../Box';
+import Flex from '../Flex';
+import StyledTabs, { Props as StyledTabProps } from './StyledTabs';
+import { Props as TabProps } from './Tab';
+import { ENTER_KEY, SPACEBAR } from '../constants';
+
+type Tab = ReactElement<TabProps>;
+
+const getActiveTab = (
+  activeTabId: string | undefined,
+  children: Array<Tab | null>
+) =>
+  children.find((tab) => {
+    if (!tab) {
+      return false;
+    }
+
+    const {
+      props: { id }
+    } = tab;
+
+    return id === activeTabId;
+  });
+
+const getTabContentId = (tab: Tab) => {
+  if (!tab) {
+    return undefined;
+  }
+
+  const {
+    props: { children, id }
+  } = tab;
+
+  return children?.props.id || `${id}-tab-content`;
+};
+
+type Props = Omit<StyledTabProps, 'children'> & {
+  activeTabId?: string;
+  children: Tab | Array<Tab | null>;
+  defaultTabId?: string;
+  tabBarAside?: ReactNode;
+};
+
+const Tabs = ({
+  activeTabId,
+  children,
+  defaultTabId,
+  tabBarAside,
+  ...props
+}: Props) => {
+  const childrenAsArray = castArray(children);
```

Comment:
Wrap any non-array in an array so we can treat a single-child as an array.  This is to address an issue in the main app when attempting to inline a single child Tab.